core/configfetcher/configfetcher.go

```diff
@@ -163,7 +165,11 @@ func (c *configFetcher) Close(ctx context.Context) (err error) {
 
 func (c *configFetcher) notifyClientApp(status *coordinatorproto.SpaceStatusPayload) {
 	s := convertToAccountStatusModel(status)
+	if c.lastStatus == s.StatusType {
```

Comment:
seems like we have a chance of concurrent access here

Reply:
You mean we need to use some kind of mutex here?

Reply:
exactly

Reply:
@deff7 I have introduced mutex. Please take a look